src/content/add-to-app/ios/project-setup.md

```diff
@@ -104,46 +99,55 @@ directory. Changes made in your module's `.ios/`
 directory don't appear in your existing iOS project
 using the module, and might be overwritten by Flutter.
 
-Do not source control the `.ios/` directory since it's autogenerated.
+Exclude the `.ios/` directory from source control as it's autogenerated.
 Before building the module on a new machine, run `flutter pub get`
-in the `my_flutter` directory first to regenerate the `.ios/`
-directory before building iOS project using the Flutter module.
+in the `my_flutter` directory first.
+This regenerates the `.ios/` directory before building iOS project
+using the Flutter module.
 :::
 
-## Embed the Flutter module in your existing application
+## Embed a Flutter module in your iOS app
 
 After you have developed your Flutter module,
 you can embed it using the methods described at the top of the page.
 
-:::note
 You can run in Debug mode on a simulator or a real device,
-and Release on a real device. Learn more about
-[Flutter's build modes][build modes of Flutter]
+and Release on a real device.
 
-To leverage Flutter debugging functionality 
-such as hot reload, see  [Debugging your add-to-app module][].
+Using Flutter [increases your app size][].
+
+:::note
+Learn more about [Flutter's build modes][build modes of Flutter].
+
+To use Flutter debugging features such as hot reload,
+consult [Debugging your add-to-app module][].
 :::
 
-Using Flutter [increases your app size][].
+### Method A - Embed with CocoaPods and the Flutter SDK {:#method-a}
```

Comment:
I've always found this "Option A/Option B/Option C" list awkward and makes the page very long. Is there any existing website doc semantics for a decision tree? Or make this tabbed so you only see one at a time?  Could use tech writer expertise for advice on how to communicate this better!

Reply:
A huge +1 to this. I would love to see tabs like on the installation guide: https://docs.flutter.dev/get-started/install/macos/mobile-ios?tab=ios16#install-the-flutter-sdk

Reply:
No issue with doing that. It does cause some issues with inline linking, but nothing too offputting.